pdns/backends/gsql/gsqlbackend.cc

```diff
@@ -1287,6 +1287,20 @@ bool GSQLBackend::createDomain(const DNSName &domain, const DomainInfo::DomainKi
       bind("account", account)->
       execute()->
       reset();
+
+    if (zoneId != nullptr) {
+      // XXX: needs its own stmt as godbc has a table name in it
```

Comment:
:exclamation:

Reply:
oh right, totally forgot about this :(

Reply:
If at some point this is the last blocker, I'm okay with noting this one in a separate ticket, so we can get 4.5.0-alpha1 out the door.

Reply:
I've added a new statement.